native/android/app/build.gradle

```diff
@@ -129,6 +129,8 @@ def jscFlavor = 'org.webkit:android-jsc:+'
 def enableHermes = project.ext.react.get("enableHermes", false);
 
 android {
+    ndkVersion rootProject.ext.ndkVersion
```

Comment:
Why did you add this. After removing the app compiles again,

Reply:
https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2
Upgrade helper says so. Seems like I forgot the `ndkVersion` in `android/build.gradle` though.

Reply:
@Taggotty could you try again please?